nidirect_sst/nidirect_sst.module

```diff
@@ -0,0 +1,6 @@
+<?php
+
+/**
+ * @file
+ * Contains nidirect_sst.module.
+ */
```

Comment:
As this file is empty, maybe you should just get rid of it ?

Reply:
Agreed, didn't spot this. Thanks